pkg/workflow/def/workflow/do_while.go

```diff
@@ -0,0 +1,81 @@
+package workflow
+
+import (
+	"fmt"
+
+	"github.com/conductor-sdk/conductor-go/pkg/http_model"
+)
+
+var (
+	LOOP_CONDITION = "loop_count"
```

Comment:
nit: we should use MixedCaps or mixedCaps for the naming:  https://go.dev/doc/effective_go#Getters

Reply:
fixed.